aws-globalaccelerator-accelerator/aws-globalaccelerator-accelerator.json

```diff
@@ -66,7 +68,8 @@
     },
     "AcceleratorArn": {
       "description": "The Amazon Resource Name (ARN) of the accelerator.",
-      "type": "string"
+      "type": "string",
+      "pattern": "^arn:(aws|aws-cn|aws-us-gov|aws-iso|aws-iso-b):(\\w+)::(\\d{12}):(accelerator)/([0-9a-f-]+)$"
```

Comment:
If we change the pattern in future APIs, we will have to make a change here as well. Is it a requirement to have regex here?

Reply:
There was no requirement. We only had the pattern in one place (EPG arn) and that was not valid according to the validation.  I updated to be valid and added pattern to all other arns.  I can remove if you feel strongly about it.

Reply:
Discussed and closed offline. We will remove this check.